depthai_examples/ros1_src/yolov4_spatial_publisher.cpp

```diff
@@ -1,6 +1,7 @@
 
 #include "ros/ros.h"
 
+#include <depthai-shared/properties/MonoCameraProperties.hpp>
```

Comment:
Include  
```
#include <depthai/pipeline/node/MonoCamera.hpp>
```

And use `dai::node::MonoCamera::Properties::SensorResolution`, etc... instead

Reply:
Fixed in b9b6b81.